manifests/set.pp

```diff
@@ -1,13 +1,13 @@
 # manage a named set
 define nftables::set(
-  Enum['ipv4_addr', 'ipv6_addr', 'ether_addr', 'inet_proto', 'inet_service', 'mark']
-    $type,
   Enum['present','absent']
     $ensure = 'present',
-  Pattern[/^[a-zA-Z0-9_]+$/]
+  Pattern[/^[-a-zA-Z0-9_]+$/]
```

Comment:
shouldn't that be escaped `\-` ?

Reply:
Hi! Typically, inside character classes the hyphen can be added at the beginning or at the end unescaped as the context is enough for the parser to remove the special meaning.

Reply:
I see, didn't know about that.